sass/package.json

```diff
@@ -1,5 +1,5 @@
 {
   "devDependencies": {
-    "sass": "1.13.4"
+    "sass": "1.14.3"
```

Comment:
This is supposed to happen automatically (e.g. https://github.com/bazelbuild/rules_sass/commit/6cd34bec2b7300bcc2dbc74da2d5d2e4e84ee35f). @nex3 do you know if something might have changed in this automation?

Reply:
It looks like we're pushing tags (https://github.com/bazelbuild/rules_sass/tree/1.14.3) but not pushing to master. I'll fix the deploy script.